phoenicis-cli/pom.xml

```diff
@@ -21,11 +21,11 @@
     <parent>
         <groupId>org.phoenicis</groupId>
         <artifactId>phoenicis</artifactId>
-        <version>${phoenicis.version}</version>
+        <version>5.0-SNAPSHOT</version>
```

Comment:
Is this required to make the OSX build work?
If not I think it would be better to move these unrelated pom changes to its own PR.

Reply:
Yes it is. Without this, the module maven-javafx will not work on a submodule

Reply:
So basically the plugin doesn't work with maven parameters (in our case the version of the module)?

Reply:
It is normally not allowed by maven to use parameters in the <parent> section

Reply:
Yes, IntelliJ complains about this all the time as well. If you think about it, it really doesn't make sense to do this either. You could very well update one module while leaving the other one untouched. Maybe we could use one of the solutions provided [here](https://stackoverflow.com/questions/17312681/maven-multi-module-project-version-management)?

Reply:
Sure. Can you do that?